README.md

```diff
@@ -204,6 +204,20 @@ reported as enabled:
 http://localhost/users/101?features.show-user-email
 ```
 
+---
+
+### Identify a new user
```

Comment:
Please add a link to this in the table of contents (I probably need to rename that to Table of Contents or something...) - https://github.com/TrueCar/react-launch-darkly#docs